VERSION-RANGE-SPEC.rst

```diff
@@ -240,6 +240,10 @@ A list of versions, enumerated:
 
 - ``vers:pypi/0.0.0|0.0.1|0.0.2|0.0.3|1.0|2.0pre1``
 
+Additionally, all versions (or none) can be specified:
+
+- ``vers:all/*``
+- ``vers:none/*``
```

Comment:
Also, we need to state somewhere that `*` is the only valid version constraint these versioning schemes can have and that no other construct is valid (even not a space in between).

Reply:
Currently, the document says ```A separate document will provide details for each versioning scheme```. Is someone already taking care of this separate document?

Reply:
I'm not aware of that effort.
@pombredanne Any insights in this?